### ISSUE TITLE
move from dev to stagging

### ISSUE THREAD

**Issue by raynamharris** [nih-cfde/update-content-registry#23]
we have a request to moving our testing from dev to stagging. will post details about what commands need to be updated to do this

**Comment by raynamharris** [comment]
1. set an environment variable `DERIVA_SERVERNAME=app-staging.nih-cfde.org` (this defaults to app-dev's FQDN in these CLI utils)
2. find the latest release id by going to app-staging.nih-cfde.org then going to "For Submitters"  >  "List All Submission" then "Submission System" > "Releases". Find the "Release Id" for the latest public release

**Comment by jessicalumian** [comment]
More info from Karl:

Karl:

First, navigate into the submission listings. Then, there is an extra navbar path available `Submission System` -> `Releases ` taking you to a list of release builds. There will be one that is showing its current status as public release, which means it is the one available via the special catalog ID "1" in the portal.  That's presumably the release you want to be refreshing.

Like the submission listings, the release listings are sorted in reverse chronological order. When I am building release candidates, but have not yet "published" them as the special catalog "1", they might appear on the top of the list above the current public release catalog. When we actually start putting the resource markdown into production, we would likely want to refresh one of these builds in the days/hours leading up to publishing the next release, i.e. if markdown has been updated in the period after I first prepared the release candidate...

Rayna:

thanks! so the release id for the june 28 release would be `f1da20a5-24c1-418e-9707-371702f3987e`

Karl:

They will be different in every build and every environment (dev/staging/prod).  The staging system's current "public release" should have all the latest DCC submissions plus a mock/test submission, I believe.

I have not yet built the next release candidate in the prod environment, so don't have a UUID for it.  I plan to wait until QA is essentially done to be sure I am building this data with whatever tagged version of cfde-deriva we will be releasing.

Oh, I forgot that if we have a "draft" release in prod, it would already have the UUID assigned.  But, a draft is just a metadata definition of which constituent submissions will be in a build, subject to change if the draft is revised prior to build. The markdown refresh functions won't work on a draft since the actual catalog/database doesn't exist yet.

(Also, if I somehow were to mess up badly enough while building a draft, my response might be to abandon it and start a new draft with a new UUID.  So the "next" release UUID isn't guaranteed to be what we'd actually release until we converge to the point of having a successful release candidate and deciding we will go ahead with publishing it...)
